src/Features/CSharp/Portable/Wrapping/InitializerExpression/AbstractCSharpInitializerExpressionWrapper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Indentation;
+using Microsoft.CodeAnalysis.Wrapping.InitializerExpression;
+
+namespace Microsoft.CodeAnalysis.CSharp.Wrapping.InitializerExpression
+{
+    internal abstract class AbstractCSharpInitializerExpressionWrapper<TListSyntax, TListItemSyntax>
+        : AbstractInitializerExpressionWrapper<TListSyntax, TListItemSyntax>
+        where TListSyntax : SyntaxNode
+        where TListItemSyntax : SyntaxNode
+    {
+        protected override string Indent_all_items => FeaturesResources.Indent_all_elements;
+        protected override string Unwrap_all_items => FeaturesResources.Unwrap_all_elements;
+        protected override string Unwrap_list => FeaturesResources.Unwrap_element_list;
+        protected override string Wrap_every_item => FeaturesResources.Wrap_every_element;
+        protected override string Wrap_long_list => FeaturesResources.Wrap_long_element_list;
+        protected override bool DoWrapInitializerOpenBrace => true;
```

Comment:
Is this "open brace needs to go on the next line?" If so, we have an option for that in C#, is this respecting that or always assuming it's checked?

![image](https://user-images.githubusercontent.com/201340/71752920-cfa40a00-2e35-11ea-823b-cfb8d512dc25.png)

Reply:
Yes, i think you're correct here @jasonmalinowski .  We should be following this formatting optino.

Reply:
(and we should explicitly have tests for this area).

Reply:
I wasn't aware that was a setting. I will change it to respect what the user has set.